docusaurus/docs/Android/04-compose/05-message-components/03-message-list.mdx

```diff
@@ -157,6 +157,30 @@ MessageList(
 
 You can also provide your own instance of `LazyListState` based on your custom logic where you define the starting scroll position of the state, the offset and how it behaves based on user events. For example, when deep linking.
 
+## Previewing Attachments
+
+Out of the box previews are provided for the following attachment types: uploading, link, Giphy, image, video and file.
+
+### Image and Video
+
+Image and video attachments are previewed as thumbnails which can be displayed as a single tile or multiple ones depending on how many attachments are contained within the specific message. By default, these are rendered by `MediaAttachmentFactory`.
+
+:::note
+Attachment factories are used for previewing attachments. You can use those we provide out of the box, add to them, or replace them with your own. You can read more about them [here](../03-general-customization/01-chat-theme.mdx#streamattachmentfactories).
```

Comment:
```suggestion
Attachment factories are used for previewing attachments. To learn more about the factories and how to customize them, click on the link [here](../03-general-customization/01-chat-theme.mdx#streamattachmentfactories).
```

More concise and simple.

Reply:
Made a slight edit that leaves out the definite article and committed your suggestion.